spec/unit/move_task_spec.rb

```diff
@@ -0,0 +1,32 @@
+# Copyright (c) 2013-2015 SUSE LLC
+#
+# This program is free software; you can redistribute it and/or
+# modify it under the terms of version 3 of the GNU General Public License as
+# published by the Free Software Foundation.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.   See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program; if not, contact SUSE LLC.
+#
+# To contact SUSE about this file by physical or electronic mail,
+# you may find current contact information at www.suse.com
+
+require_relative "spec_helper"
+
+describe MoveTask do
+  let(:move_task) { MoveTask.new }
+
+  describe "#move" do
+    it "moves the system description directory" do
+      store = double
+      from = "description1"
+      to = "description2"
+      expect(store).to receive(:move).with(from, to)
+      move_task.move(store, from, to)
```

Comment:
Do we have tests for the methods which check for the [non]existance of system descriptions?

Anyway I would at least add one test for a conflict I think.